app/models/katello/concerns/pulp_database_unit.rb

```diff
@@ -67,9 +67,17 @@ def import_all(uuids = nil, options = {})
         end
       end
 
-      def import_for_repository(repository, force = false)
+      def import_for_repository(repository, force = false, full_index = false)
```

Comment:
I think we should just combine force and full_index into one argument.  They kinda have the same meaning (skill optimizations).